src/main/resources/template/select.vm

```diff
@@ -0,0 +1,95 @@
+#* @vtlvariable name="action" type="com.atlassian.shele.shelePlugin.webwork.SelectAction" *#
+
+#set($ft = "Jira Loads ")
+#set($sc="Jira класс?")
+#set($title="Okay bro!")
+#set($context=$action.servletContext)
+<html>
+<head>
+  <meta name="decorator" content="atl.admin"/>
+  <meta name="admin.active.section" content="admin_plugins_menu/my-section"/>
+  <meta name="admin.active.tab" content="my-next-item"/>
+  <script src="https://connect-cdn.atl-paas.net/all.js"></script>
+  <title>$title</title>
+</head>
+<body>
+<form class="aui" action="http://localhost:2990/jira/rest/rest-issue/1.0/api/issue" method="post" id="form-1">
```

Comment:
what are you going to do on the production?
please do not hardcode the domain URL - `http://localhost:2990`! it should be defined by context!